tests/system/python/packages/test_multiple_assets.py

```diff
@@ -313,24 +325,26 @@ def test_multiple_assets_with_reconfig(self, reset_fledge, start_north, read_dat
                 on endpoint GET /fledge/asset"""
 
         total_benchmark_services = 3
-        num_assets = 2 * PER_BENCHMARK_ASSET_COUNT
-        # Total number of assets that would be created
-        total_assets = PER_BENCHMARK_ASSET_COUNT * total_benchmark_services
+        num_assets_per_service = (num_assets//(total_benchmark_services*2))
```

Comment:
what is * 2

Reply:
Here "* 2" is used to divide the number of assets per services properly.
For e.g.,
- We want to create total 600 assets
- Initially, we will create 3 services and each service will create 100 assets.
- After reconfig, in each service more 100 assets will be added.
- At the end of test total 600 assets will be created by each service.